.travis.yml

```diff
@@ -2,14 +2,22 @@ language: go
 
 python: 2.7
 
+services:
+  - mysql
+
 before_install:
   - pip install --upgrade pip virtualenv
   - virtualenv "${HOME}/FSENV"
+  - mysql --print-defaults
 
 install:
   - source "${HOME}/FSENV/bin/activate" && pip install -e .
   - go get -v -t ./... # Install dependencies needed for tests.
 
+before_script:
+  - export MYSQL_TEST_USER=travis
```

Comment:
Use the 'env' attribute to set env variables e.g. see https://github.com/google/grr/blob/master/.travis.yml